docs/data/base/components/switch/switch.md

```diff
@@ -22,23 +21,32 @@ To make the switch accessible, you should ensure the corresponding label reflect
 import SwitchUnstyled from '@mui/base/SwitchUnstyled';
 ```
 
-The `SwitchUnstyled` component provides default components and assigns CSS classes you can style entirely on your own.
-You are free to choose any styling solution - plain CSS classes, a CSS framework, Emotion, etc.
-It is also possible to replace these default components by other HTML elements or custom components.
+## Basic switch
 
-There are three components you can override by the `components` prop: `Root`, `Thumb` and `Input`. Each one's props can be set using the `componentsProps` object.
+The `SwitchUnstyled` component is composed of three interior components—`Root`, `Thumb`, and `Input`—and it assigns CSS classes for styling each piece, respectively.
```

Comment:
The demo below also applies styles to a `track`. Does that mean there are actually four interior components?

Reply:
Seeing the HTML structure yes.

![image](https://user-images.githubusercontent.com/4512430/165527775-cc79d040-6c7a-4c5b-a29a-77c4bdefe46e.png)

Reply:
I see, so the track, thumb, and input are all nested within the root. I'll revise this line to make that more clear.